docs/SPEC.md

```diff
@@ -1487,7 +1487,7 @@ Covariance has the following properties:
 
 | Name     | Type     | Description                                                                                 |
 | ----     | ----     | -----------                                                                                 |
-| columns  | []string | Columns specifies a list of columns to aggregate. Defaults to `["_value"]`.                 |
+| columns  | []string | Columns specifies a list of the two columns to aggregate. This property is required and has no default.                 |
```

Comment:
I would say that, as such, `covariance` is no more an aggregate? Because it cannot be used in `aggregateWindow`. Should we provide a `covarianceBuilder` that given two columns, provides an aggregate? Like a function generator, idk 🤔

Reply:
```
covGen(col1, col2) = (column: col1) => covariance(columns: [col1, col2])
```

Reply:
I agree.  but I didn't know how else to categorize it.  in one way, it is an aggregator because it is combining points into a single value, but in the other way it is not because its function signature is different.  I thought about using a function generator pattern to make it fit, but I couldn't think of a good reason to hard-code one parameter.  

something like  `covGen(column: col1) = (column: col2) => covariance(columns: [col1, col2])` could work but the user is free to create this function if they need it.